### ISSUE TITLE
Meqtrees master no longer passes tests

### ISSUE THREAD

**Issue by bennahugo** [ska-sa/meqtrees#878]
I think there has been a change somewhere. Master is failing. 
```
copying Cattery/Scripter/scripter.10calibration.funcs -> /usr/local/lib/python2.7/dist-packages/Cattery/Scripter
running install_egg_info
Writing /usr/local/lib/python2.7/dist-packages/meqtrees_cattery-1.5.1.egg-info
 ---> 9f00521aaf2b
Removing intermediate container ac7bea4f7563
Step 7/8 : WORKDIR /code/test/Batchtest
 ---> 55e32deb73e0
Removing intermediate container 53eee127223e
Step 8/8 : CMD python batch_test.py
 ---> Running in 0257308f11d3
 ---> e5a876b78949
Removing intermediate container 0257308f11d3
Successfully built e5a876b78949
Successfully tagged catterycron:41
+ docker run --rm catterycron:41
Can't locate owlcat or owlcat.sh
Build step 'Execute shell' marked build as failure
Sending e-mails to: oms@ska.ac.za
Triggering a new build of Docker-killswitch
Finished: FAILURE
```

**Comment by bennahugo** [comment]
@o-smirnov looks like owlcat is missing from the dependencies?

**Comment by bennahugo** [comment]
Last successful test: Nov 19, 2017 3:00 PM

**Comment by o-smirnov** [comment]
@gijzelaerr have you changed the pip dependencies of ``meqtrees``? Looks like it no longer pulls in ``owlcat``, which makes the test fail.

**Comment by bennahugo** [comment]
Exact steps to reproduce:
```
IMAGE=catterycron:$BUILD_NUMBER
docker build -t $IMAGE .
docker run --rm $IMAGE
```
Output see: https://jenkins.meqtrees.net/job/meqtrees-cattery-cron/43/console

**Comment by gijzelaerr** [comment]
nope: that file didn't change in a while:

https://github.com/ska-sa/meqtrees-cattery/commits/master/setup.py

maybe a dependency changed dependencies? But if meqtrees always depend on owlcat it should just be added ofcourse.

**Comment by bennahugo** [comment]
Fixed with ska-sa/meqtrees-cattery#77
